src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC } from 'react';
+import { Person } from '../../types';
+import { PersonItem } from '../PersonItem';
+
+type Props = {
+  people: Person[];
+  selectedSlug: string;
+};
+
+export const PeopleTable: FC<Props> = ({ people, selectedSlug }) => {
+  const isNoPeopleOnServer = people.length === 0;
+
+  if (isNoPeopleOnServer) {
+    return (
+      <p data-cy="noPeopleMessage">
+        There are no people on the server
+      </p>
+    );
+  }
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const mother = people
+            .find(personItem => personItem.name === person.motherName);
+          const father = people
+            .find(personItem => personItem.name === person.fatherName);
+
```

Comment:
will be better to add these objects to each person before calling `map` function
for now it'll be called for each rerendering, will be better to call it only one after receiving people from server